src/FluxCalculator.jl

```diff
@@ -289,15 +291,18 @@ function surface_thermo_state(
     sim::Interfacer.SurfaceModelSimulation,
     thermo_params::TD.Parameters.ThermodynamicsParameters,
     thermo_state_int,
-    colidx::Fields.ColumnIndex,
+    colidx::Fields.ColumnIndex;
+    ΔT_sfc = 0,
 )
+    FT = eltype(parent(thermo_state_int))
     @warn("Simulation " * Interfacer.name(sim) * " uses the default thermo (saturated) surface state", maxlog = 10)
-    T_sfc = Interfacer.get_field(sim, Val(:surface_temperature), colidx) #
+    T_sfc = Interfacer.get_field(sim, Val(:surface_temperature), colidx) .+ FT(ΔT_sfc)
```

Comment:
Do we need an "updated" surface temperature here, but only the non updated one is available? I don't quite understand what's happening here but a comment could help others interpret this line.

Reply:
ohh gotcha